WolvenKit.App/ViewModels/GraphEditor/RedGraph.Quest.cs

```diff
@@ -458,18 +472,23 @@ public static RedGraph GenerateQuestGraph(string title, graphGraphDefinition que
         var nodeCache = new Dictionary<uint, BaseQuestViewModel>();
         foreach (var nodeHandle in questGraph.Nodes)
         {
-            ArgumentNullException.ThrowIfNull(nodeHandle.Chunk);
-
-            var node = nodeHandle.Chunk;
-            var nodeID = 0;
+            var node = nodeHandle.GetValue();
+            if (node is null)
+            {
+                throw new Exception("Empty nodes aren't supported!");
```

Comment:
Maybe you want to catch this exception somewhere further up, and only print the message? Because exceptions are scary to users

Reply:
We already do that?
I just added/changed the text so its more clear what is wrong